src/utils/URLParamsParser.ts

```diff
@@ -0,0 +1,6 @@
+export function paramsParser(queryParams: Record<string, string>) {
+	for (let i in queryParams) {
```

Comment:
Try sending multiple items in the queryParams object and verify whether you get the expected result.

Reply:
current implemention will take a singe param which will append to the url with "?" in front of the param. For multiple params we can chain them using '&'